training/README.rst

```diff
@@ -0,0 +1,163 @@
+software, Odoo user basic training*, etc.
+
+They are classified with action types. Depending on it, they define how many
+hours of each type are expected to be teached.
+
+You can define the training materials that should be delivered to students to
+accomplish the action.
+
+You can specify the grade settings: which is the minimal and maximum grade
+possible for this course, and which grade students need to reach to pass the
+course.
+
+Also you can define the contents of this course. They will appear in the back
+side of the diploma if available. You have a field called *Append template*
+that will help you to fulfill the contents in some complex layouts.
+
+Action Types
+~~~~~~~~~~~~
+
+Depending on the type of hours that a training action is expected to have, the
+training action has that action type. For instance:
+
+- A training of type *on-site* may expect only *on-site* hours.
+- A training of type *online* may expect only *online* hours.
+- A training of type *remote* may expect only *remote* hours.
+- A training of type *mixed* may expect *on-site*, *online* and *remote* hours.
+
+You can define action types and the expected type of hours in this menu.
+
+Hour Types
+~~~~~~~~~~
+
+Hours in each training action are separated by type. Types of hours can be *On
+site, Online, Remote* and anything you decide.
+
+For instance, a training action of *online* type cannot have *on-site* hours.
+
+Materials
+~~~~~~~~~
+
+A material is a resource used for training. These resources are expected to be
+delivered to every student when starting the training process.
+
+Example materials are: study book, notebook, pen, etc.
+
+Material types
+~~~~~~~~~~~~~~
+
+Materials are not always physical. For example, for online courses a *material*
+could be a username and password for your training website.
+
+In this menu you can set up all the types of your materials.
+
+Groups
+~~~~~~
+
+A training group is a course that belongs to a training action. Most of the
+time you will likely use this menu.
+
+They have all details about it: students, start and end dates, materials
+delivered, etc.
+
+Technically speaking, groups are just events that are assigned to a training
+action. Therefore, you can use also the events view to manage training groups,
+but the *Training* menu is a more consistant place to manage training groups,
+and when using that view you will be required to set a training action.
+
+Registrations (inside Groups form view)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+These are similar to event registrations too, but with some additions:
+
+- Set a grade for the student and see if it is passing the course
+  automatically (it compares with the passing grade in the training action).
+- Quick access to the training action.
+- Check if materials were deliverede to the registrant.
+
+There are also these reports here:
+
+- *Diploma / Certificate*. If the attendee has completed and passed the
```

Comment:
Printed diploma to be delivered if the attendee has completed and passed the course. Otherwise a certificate of assistance will be delivered